WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -369,12 +369,16 @@ - (BOOL)fb_dismissKeyboardWithKeyNames:(nullable NSArray<NSString *> *)keyNames
     if (nil != auditTypeValue) {
       auditType = valuesToNamesMap[auditTypeValue] ?: [auditTypeValue stringValue];
     }
+    id<FBXCElementSnapshot> snapshot = [extractIssueProperty(issue, @"element") fb_takeSnapshot];
```

Comment:
Please assign the result of `extractIssueProperty(issue, @"element")` to a variable so you don't have to call it multiple times